pkg/test/framework/components/istio/operator.go

```diff
@@ -297,6 +341,16 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 		if c.environment.IsControlPlaneCluster(cluster) {
 			// Expose Istiod through ingress to allow remote clusters to connect
 			installSettings = append(installSettings, "--set", "values.global.meshExpansion.enabled=true")
+			if c.environment.Settings().CentralIstiod {
+				installSettings = append(installSettings, "--set", "values.global.centralIstiod=true",
+					"--set", "values.global.caAddress="+"istiod.istio-system.svc:15012",
+					"--set", "values.gateways.istio-ingressgateway.meshExpansionPorts[0].port=15017",
+					"--set", "values.gateways.istio-ingressgateway.meshExpansionPorts[0].targetPort=15017",
+					"--set", "values.gateways.istio-ingressgateway.meshExpansionPorts[0].name=tcp-webhook",
+					"--set", "values.gateways.istio-ingressgateway.meshExpansionPorts[1].port=15012",
+					"--set", "values.gateways.istio-ingressgateway.meshExpansionPorts[1].targetPort=15012",
+					"--set", "values.gateways.istio-ingressgateway.meshExpansionPorts[1].name=tcp-istiod")
+			}
```

Comment:
is it possible to feed a yaml instead?  it is unusual for user to have these many set params.

Reply:
cfg.ControlPlaneValues, but that is applied to every cluster.